drivers/ethernet/eth_native_posix_adapt.c

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ *
+ * Routines setting up the host system. Those are placed in separate file
+ * because there is naming conflicts between host and zephyr network stacks.
```

Comment:
Which ones are conflicting?

Reply:
There are network related symbols (like struct sockaddr_in6) in include/net/net_ip.h that will conflict with host /usr/include.